_notices/rsn0029.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 29 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Dropping Python v3.8 in v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"

```

Comment:
How do we decide between "23.06" and "23.06+"? Is dropping support something that affects future versions, thus deserving the "plus"? I'm not sure if I can tell how this was decided in past RSNs: https://docs.rapids.ai/notices/rsn
```suggestion
notice_rapids_version: "v23.06+"

```